applications/openshift/general/file_integrity_notification_enabled/tests/ocp4/e2e.yml

```diff
@@ -0,0 +1,3 @@
+---
+default_result: FAIL
+result_after_remediation: PASS
```

Comment:
nice to see this passes. Where's the remediation though? I'm guessing this relies on the manual remediation we already have that installs FIO, does it?

Reply:
Yeah it does, `applications/openshift/integrity/file_integrity_exists/tests/ocp4/e2e-remediation.sh`